services/launchpad/launchpad.tester.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('../service-tester');
+
+const t = new ServiceTester({ id: 'launchpad', title: 'Launchpad.net' })
+module.exports = t;
+
+t.create('gimp ppa version')
+  .get('/ppa/version/otto-kesselgulasch/gimp/gimp.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'version',
+    value: Joi.string()
+  }));
+
+t.create('gimp ppa downloads')
+  .get('/ppa/downloads/otto-kesselgulasch/gimp/gimp.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'downloads 24h',
+    value: Joi.number()
```

Comment:
What do you think about using `isMetric` validator here? https://github.com/badges/shields/blob/2ded83f8a68cd36102167883d0f9008a7404cb24/services/test-validators.js#L53?